### PR TITLE
Migrate from Travis CI to GitHub Actions

### PR DESCRIPTION
**Pull request by lpeabody**
# Changes

- Migrate to GitHub Actions.

### CODE REVIEW

**Comment by lpeabody** [comment]
https://github.com/docksal/service-mysql/runs/2453371055?check_suite_focus=true#step:9:122

The error here doesn't make sense to me. mysql-variables.txt looks identical to `SHOW VARIABLES;`.

**Comment by lpeabody** [comment]
If you have BATS installed locally and you run:

```
export VERSION=8.0 FROM=mysql:8.0
make
make test
```

It succeeds:

```
IMAGE=docksal/mysql:8.0 NAME=docksal-mysql-8.0 VERSION=8.0 ./tests/test.bats
 ✓ Container initialization
 ✓ Default database present
 ✓ Check variables
 ✓ Configuration overrides

4 tests, 0 failures
```

EDIT: Nevermind, it actually fails. I forgot to rerun `make` after switching from version 5.6 to running version 8.0 (so I didn't follow the steps above the first time... and then I followed the steps above and got the error).

**Comment by lpeabody** [comment]
According to https://stackoverflow.com/a/30074553/664881 `utf8` is an alias for `utf8mb3`, which is the value you get when you query the variables on version 8 of the built container.

Also see https://dev.mysql.com/doc/refman/8.0/en/charset-unicode-utf8mb3.html. Seems to suggest this should switch to utf8mb4 from utf8mb3 since utf8mb3 is deprecated.

File: 8.0/mysql-variables.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ character_set_database	utf8mb4
 character_set_filesystem	binary
 character_set_results	utf8mb4
 character_set_server	utf8mb4
-character_set_system	utf8
+character_set_system	utf8mb3
```
Comment (lmakarov) [inline 8.0/mysql-variables.txt]: Per 
- https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_character_set_system
- https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_character_set_system 

`character_set_system` is always `utf8` (default). As such, we should just drop it altogether.

**Review by lmakarov — APPROVED** [review]

**Pull request merged**